packages/webapp/pages/squads/new.tsx

```diff
@@ -92,6 +93,8 @@ const NewSquad = (): ReactElement => {
         initialData={{
           name: data?.name,
           handle: initialHandle,
+          memberPostingRole: SourceMemberRole.Member,
```

Comment:
This is already set on API no?
https://github.com/dailydotdev/daily-api/blob/cc174bcecdad2cac05371d2e1be854337bc5aea5/src/schema/sources.ts#L1923

Reply:
Yeah, but in FE, the default state is at `false`.